core/app/views/layouts/refinery/admin.html.erb

```diff
@@ -2,22 +2,30 @@
 <%= render '/refinery/html_tag' %>
   <% content_for :meta, tag(:meta, name: 'refinerycms', content: Refinery.version) %>
   <%= render 'refinery/admin/head' %>
-  <body class="<%= action_name %> <%= I18n.locale %>">
+  <body class="<%= action_name %> <%= I18n.locale %> cms-interface">
     <%= render 'refinery/admin/menu' %>
     <%= render '/refinery/site_bar' %>
     <div id='tooltip_container'></div>
-    <div id="admin_container" class="clearfix">
-      <div id="page_container">
-        <div id="page">
-          <div id="content" class="clearfix">
-            <div id="flash_container">
-              <%= render '/refinery/no_script' %>
-              <%= render '/refinery/message' %>
+    <section id="wrapper" class="dimmer">
+      <% if content_for?(:admin_content) %>
```

Comment:
This feature, as I said in the initial comment is "Supporting css & partials for glass admin index views. The first one is here: refinery/refinerycms-authentication-devise#23".  If you look at that other PR, I have a screenshot that shows the new look.  I don't like the idea of any test for styles, so if all current tests pass without regression I think that's all we can expect for this one.

I removed the standard `yield` as a simple way to have old and new layouts working at the same time and I simply don't return anything with the standard yeild whenever I use the `admin_content`.  If there is a better way for a view to change a layout condition other than `if content_for?` we can use that.  I thought this code read well.

It may help to see where I'm going with this too.  Here's what's to come if we get here.  This includes 3 sections in the admin interface: A right sidebar (somewhat like the left sidebar with the nav), a page preview (this would yield a frontend `show` view to edit within), and then `admin_content` (index views, forms that don't make use of the preview).

```
    <% if content_for?(:admin_sidebar) %>
      <section id="sidebar-right" class="sidebar right off-canvas-sidebar cms-ui">
        <div class="sidebar-top-nav">
           <span>Publish settings</span>
           <button type="button" class="btn sidebar-right-close"><i class="icon icon-close"></i></button>
        </div>
        <div id="sidebar-right-inner" class="sidebar-inner">
          <%= yield :admin_sidebar %>
        </div>
      </section>
    <% end %>

    <section id="wrapper" class="dimmer dimmed-editing">
      <% if content_for?(:admin_page_preview) %>
        <section id="page-preview">
          <div class="page-container">
            <%= yield :admin_page_preview %>
          </div>
        </section>
      <% elsif content_for?(:admin_content) %>
        <div class="cms-ui">
          <%= yield :admin_content %>
        </div>
      <% end %>
    </section>
```